Makefile

```diff
@@ -0,0 +1,9 @@
+install: install-glide
+	( cd maptime ; glide install )
+
+install-glide:
+		@which glide > /dev/null || (go get -u github.com/Masterminds/glide && cd $(GOPATH)/src/github.com/Masterminds/glide && git checkout v0.12.3 && go install)
+		@glide -version > /dev/null || (echo "Glide install failed" && exit 1)
+
+test:
+	go test ./maptime
```

Comment:
nit: could you add coverage, and race checks (can-reuse the stuff from ci-scripts). 

Maybe good to add metalinting too